test/RBdata.sql

```diff
@@ -1486,7 +1486,7 @@ UNLOCK TABLES;
 SET SQL_MODE=@OLD_SQL_MODE;
 
 DELETE FROM `mri_scan_type`;
-INSERT INTO `mri_scan_type` (ID,Scan_type)  VALUES
+INSERT INTO `mri_scan_type` (ID,MriScanTypeID)  VALUES
```

Comment:
```suggestion
INSERT INTO `mri_scan_type` (ID,Name)  VALUES
```
I think the second field field should be `Name`, not `MriScanTypeID`